### DISCUSSION TITLE
Change fontsize for footnotes?

### DISCUSSION THREAD

**Discussion by kcho17** [jgm/pandoc#9804] [Q&A] [answered]
Hi there! I am using pandoc to produce pdf. I am wondering if there's a way to change font size for my footnotes. Thanks!

**Comment by iandol** [comment]
That depends on the PDF engine you are using. If you are using a LaTeX path, then you will need to edit the LaTeX template to do this:

https://pandoc.org/MANUAL.html#templates

...and implement something like https://tex.stackexchange.com/questions/249415/set-font-size-for-footnotes

You can use YAML by using the template syntax, so for example if you were using Typst for PDF generation:

```typst
$if(footnotesize)$
#show footnote.entry: set text(size: $footnotesize$)
$endif$
```

Pandoc will insert the value if it finds the `footnotesize` variable in the metadata...

**Reply by kcho17** [reply]
Thanks for this answer. I am using xelatex for the PDF engine. I try to the code you gave, but it didn't work. It doesn't need to be YAML but in any case I would like to change the fontsize for footnotes.

**Reply by kcho17** [reply, marked as answer]
Oh I think I've figured out! I just added the following in the YAML metadata
```
---
header-includes: |
	\renewcommand{\footnotesize}{\fontsize{11pt}{13pt}\selectfont}
...
```
It works!
